jekyll/_cci2/release/releases-overview.adoc

```diff
@@ -0,0 +1,242 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Releases overview - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn CircleCI releases for visibility into Kubernetes releases, how it works, and how to get set up
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
+
+Visualise and control deployments to your Kubernetes cluster with CircleCI release environments.
+
+[#introduction]
+== Introduction
+
+CircleCI release environments give you visibility into your Kubernetes deployments in the CircleCI web app, as well as access to controls for release management. From the releases dashboard, you can:
+
+* Restore a version
+* Scale a version
+* Restart a version
+
+CircleCI releases supports link:https://argoproj.github.io/argo-rollouts/[Argo Rollouts] for progressive delivery. Some further controls are available from the releases dashboard for projects that use Argo Rollouts:
+
+* Retry release
+* Promote release
+* Cancel release
+
+[#quickstart]
+== Quickstart
+
+If you want to get started with CircleCI releases right away, you can jump to:
+
+* xref:set-up-a-release-environment.adoc#[Tutorial 1: Set up a release environment]
+* xref:configure-your-kubernetes-components.adoc#[Tutorial 2: Configure your Kubernetes components]
+
+#Add links to how to guide for using commands?#
+
+[#how-the-feature-works]
+== How the feature works
+
+CircleCI integrates with Kubernetes deployments and Argo Rollouts to provide visibility and control over your release process through the CircleCI web app.
+
+#Add some information about how it all works#
```

Comment:
@ryanbarkley wondering if you can help me out with this part. Something high level about what happens under the hood when customers use the commands for example? Maybe there's something in confluence I can use to create a section here. This would be where any high level system diagrams could go too